cognitive_mapping_and_planning/tfcode/vision_baseline_lstm.py

```diff
@@ -196,6 +197,7 @@ def combine_setup(name, combine_type, embed_img, embed_goal, num_img_neuorons=No
       out = embed_goal
     else:
       logging.fatal('Undefined combine_type: %s', combine_type)
+      raise ValueError('Undefined combine_type: %s', combine_type)
```

Comment:
@s-gupta there are several more instances of `logging.fatal` in `cognitive_mapping_and_planning`, maybe worth another PR to address these

@martinwicke I see tensorflow wrapper also defines `tf.logging.fatal`, maybe this should be discussed too?